TypeScript compilation fails for WebWorker

Compiling `comlink.d.ts` for a WebWorker fails with
```
Using tsc v2.8.3
node_modules/comlinkjs/comlink.d.ts(26,23): error TS2304: Cannot find name 'Window'.
node_modules/comlinkjs/comlink.d.ts(29,44): error TS2304: Cannot find name 'Window'.
```
My workaround is to remove "Window |" from method definitions for `proxy` and `expose`.

Dont we have another option?

--- a/comlink.ts
+++ b/comlink.ts
@@ -12,7 +12,6 @@
  */
 
 /// <reference lib="dom" />
-/// <reference lib="webworker" />
 
 export interface Endpoint {
   postMessage(message: any, transfer?: any[]): void;

--- a/comlink.ts
+++ b/comlink.ts
@@ -11,6 +11,9 @@
  * limitations under the License.
  */
 
+/// <reference lib="dom" />
+/// <reference lib="webworker" />
+
 export interface Endpoint {
   postMessage(message: any, transfer?: any[]): void;
   addEventListener(

